superset/db_engine_specs.py

```diff
@@ -280,8 +280,10 @@ def get_configuration_for_impersonation(cls, uri, impersonate_user, username):
         return {}
 
 
-class PostgresEngineSpec(BaseEngineSpec):
-    engine = 'postgresql'
+class PostgresBaseEngineSpec(BaseEngineSpec):
+    """ Abstract class for Postgres 'like' databases """
+
+    engine = 'postgresqlbase'
```

Comment:
Shouldn't this be empty so people extending it have to remember to add something sensible?